classes/PodsData.php

```diff
@@ -1856,12 +1844,7 @@ public static function table_alter( $table, $changes ) {
 	 */
 	public static function table_truncate( $table ) {
 
-		/**
-		 * @var $wpdb wpdb
-		 */
-		global $wpdb;
-
-		$sql = "TRUNCATE TABLE `{$wpdb->prefix}" . self::$prefix . "{$table}`";
+		$sql = "TRUNCATE TABLE `" . self::get_pods_prefix() . "{$table}`";
```

Comment:
:no_entry_sign: String "TRUNCATE TABLE `" does not require double quotes; use single quotes instead

<sub>hash: 72d7b496986a9ab9d6584c2160248e71</sub>